### PR TITLE
[Fix]Fix the extension mysql_to_doris  bug

### LINKED ISSUE #13722: [Bug] extension mysql_to_doris: mysql:ERROR 1103 (42000) at line 1: Incorrect table name ''
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-doris/issues?q=is%3Aissue) and found no similar issues.

### Version

master

### What's Wrong?

Using extension mysql_to_doris ,there was an error executing the  e_mysql_to_doris.sh  script. 
ERROR 1103 (42000) at line 1: Incorrect table name ''.
uTools_1666851065089

### What You Expected?

e_mysql_to_doris.sh: Some of the shell command in this script is wrong.
I can submit PR to fix this bug.

### How to Reproduce?

1、To configure mysql.conf and doris.conf in the conf directory,the conf including host、port and password

2、To configure mysql_tables and doris_tables in the conf directory,the conf is user need to synchronization tables and want to get table name

3、To execute e_mysql_to_doris.sh by sh e_mysql_to_doris.sh

  ERROR 1103 (42000) at line 1: Incorrect table name ''

### Anything Else?

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by TsubameKoyasu**
e_mysql_to_doris.sh: command error,This error causes script execution errors.  :ERROR 1103 (42000) at line 1: Incorrect table name ''.
 ` ` symbol position error.

Issue Number: close #13722

## Problem summary
Using extension mysql_to_doris ,there was an error executing the e_mysql_to_doris.sh script.
ERROR 1103 (42000) at line 1: Incorrect table name ''.

Describe your changes.
fix ` ` symbol position error.

1. Does it affect the original behavior: 
    - [x] Yes
    - [ ] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [x] No Need
3. Has document been added or modified:
    - [x] Yes
    - [ ] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [x] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [x] No

### CODE REVIEW

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 38.18 seconds
 load time: 569 seconds
 storage size: 17154709829 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20221027161417_clickbench_pr_34668.html

File: extension/mysql_to_doris/bin/e_mysql_to_doris.sh [hunk 1]
```diff
@@ -31,8 +31,8 @@ rm -f $path
 #get create table sql for mysql
 for table in $(cat ../conf/mysql_tables |grep -v '#' | awk -F '\n' '{print $1}')
         do
-        d_d=`echo $table` |awk -F '.' '{print $1}'
-        d_t=`echo $table` |awk -F '.' '{print $2}'
+        d_d=`echo $table |awk -F '.' '{print $1}'`
+        d_t=`echo $table |awk -F '.' '{print $2}'`
```
Comment (adonis0147) [inline extension/mysql_to_doris/bin/e_mysql_to_doris.sh:35]: It is better to use this syntax.
```suggestion
        d_d=$(echo $table |awk -F '.' '{print $1}')
        d_t=$(echo $table |awk -F '.' '{print $2}')
```

Comment (adonis0147) [reply extension/mysql_to_doris/bin/e_mysql_to_doris.sh:35]: BTW, some checks failed. Please rebase the code to trigger them again.

Comment (TsubameKoyasu) [reply extension/mysql_to_doris/bin/e_mysql_to_doris.sh:35]: OK,That's right

**Review by adonis0147 — APPROVED** [review]
LGTM

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 38.9 seconds
 load time: 590 seconds
 storage size: 17154810757 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20221028021619_clickbench_pr_35018.html

**Pull request merged**